package-lock.json

```diff
@@ -16301,4 +16301,4 @@
       "dev": true
     }
   }
-}
+}
```

Comment:
not sure this is an intended change?

Reply:
No. Please help me, @joelanman — I commited the file by accident and attempted to revert. :(